src/main/java/storagesystem/services/JSONHandler.java

```diff
@@ -0,0 +1,278 @@
+package storagesystem.services;
+
+import com.fatboyindustrial.gsonjodatime.Converters;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import storagesystem.model.*;
+
+
+import java.io.*;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * @author Carl Lindh
+ * JSONHandler is a handler with static methods with which it is possible to write and read from the json files corresponding to the Strings declared.
+ */
+
+public class JSONHandler {
+
+    private static String itemDB = "src/main/resources/json/itemDB.json";
+    private static String locationDB = "src/main/resources/json/locationDB.json";
+    private static String organisationDB = "src/main/resources/json/organisationDB.json";
+    private static String reservationDB = "src/main/resources/json/reservationDB.json";
+    private static String teamDB = "src/main/resources/json/teamDB.json";
+    private static String userDB = "src/main/resources/json/userDB.json";
+
+    /**
+     * addToJson() automatically adds an Object to the corresponding json file without erasing its contents, if it exists.
+     *
+     * @param objectToAdd
+     * @throws IOException
+     */
+    public static void addToJson(Object objectToAdd) throws IOException {
+        String fileName;
+        if (objectToAdd instanceof Item) {
```

Comment:
instanceof är lite fult, men kanske måste kanske användas. Går det åtminstone att byta ut till interfaces?

Reply:
Har försökt, lyckas inte lösa det på något bra sätt då fileName måste vara annorlunda beroende på vilken typ som sparas.

Reply:
Removed most methods and ALL instanceof in new push